Dockerfile

```diff
@@ -30,9 +30,12 @@ COPY --from=symbolicator-planner /work/recipe.json recipe.json
 # Build only the dependencies identified in the `symbolicator-planner` image
 RUN cargo chef cook --release --features=${SYMBOLICATOR_FEATURES} --recipe-path recipe.json
 
+RUN rustup toolchain install nightly
+
 COPY . .
 RUN git update-index --skip-worktree $(git status | grep deleted | awk '{print $2}')
-RUN cargo build --release --features=${SYMBOLICATOR_FEATURES}
+RUN cargo +nightly run -Z build-std --target x86_64-unknown-linux-gnu
```

Comment:
I believe these rustup & cargo invocations should be above the `RUN cargo chef` command.

The `cargo chef cook` builds all the dependencies independently from the main symbolicator for docker layer caching purposes. Those dependencies should also be built with the proper RUSTFLAGS and std.
Plus, the newly built std should also be part of the docker layer cache.